sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/CodeGenInternal/AuthenticationTypeEnum.cs

```diff
@@ -3,7 +3,6 @@
 
 namespace Azure.AI.MetricsAdvisor.Models
 {
-    // TODO: expose it as part of 1.0.0-beta.4
```

Comment:
Explained in other comments, but we decided to not expose the `AuthenticationTypeEnum` type.